templates/welcome_email.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <h3>Hi {{firstName}},</h3>
+    <p>Thanks for filling out the form on <a href="https://www.rememberthee.com" target="_blank">rememberthee.com</a>.
+        Remember Thee is available for download on iOS only. <br>You will receive an email from Apple's app testing
+        product, <a href="https://testflight.apple.com" target="_blank">TestFlight</a>, with instructions on how to
+        download Remember Thee shortly.</p>
+
+    <p>A few things to keep in mind while downloading and using Remember Thee:</p>
+    <ul>
+        <li>This version of Remember Thee does not contain all the features you saw on the website.
+            <br>They will be added in the coming days and weeks. If there was a feature you were particularly excited
+            about, please let me know.
+        </li><br>
+        <li>In general, if there are things you wish the app could do, or if it's not really working for you, please let
+            me know.
+        </li><br>
+        <li>If the app crashes, something looks visibly broken or doesn't work or is difficult to use, or you're having
+            trouble downloading the app. <br>
```

Comment:
Please remove the `<br>` - this list item is one (long) sentence, so should be a comma after the word "app", followed by "you can reach out...."